guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -40,9 +44,10 @@ ifeval::["{context}" == "capsule"]
 ----
 +
 endif::[]
+ifeval::["{build}" == "satellite"]
 NOTE: If you are installing {ProductName} as a virtual machine hosted on Red{nbsp}Hat Virtualization (RHV), you must also enable the *Red{nbsp}Hat Common* repository, and install RHV guest agents and drivers. For more information, see https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.3/html/virtual_machine_management_guide/installing_guest_agents_and_drivers_linux#Installing_the_Guest_Agents_and_Drivers_on_Red_Hat_Enterprise_Linux[Installing the Guest Agents and Drivers on Red Hat Enterprise Linux] in the _Virtual Machine Management Guide_ for more information.
-
 +
+endif::[]
```

Comment:
This is a valid paragraph, just replace RH Virtualization with our variable which renders as oVirt upstream.